src/simulator/data-logging-hooks.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode, useContext, useEffect } from "react";
+import useRafState from "../common/use-raf-state";
+import { useSimulator } from "../device/device-hooks";
+import { EVENT_LOG_DATA } from "../device/simulator";
+
+export interface DataLog {
+  headings: string[];
+  data: DataLogRow[];
+}
+
+export interface DataLogRow {
+  isHeading?: boolean;
+  data: string[];
+}
```

Comment:
These are also defined identically in and exported from `simulator.ts`.

Reply:
Oops, I intended to move it there when I moved the log management there for performance reasons.

Reply:
3360b3f57759fb27bb5224cb61db201fd37e658a